src/main/java/org/zalando/nakadi/service/timeline/TimelineService.java

```diff
@@ -129,12 +148,26 @@ public Timeline getTimeline(final EventTypeBase eventType) throws TimelineExcept
         }
     }
 
+    public Timeline getFakeTimeline(final EventType eventType) {
+        try {
+            final Storage storage = storageDbRepository.getStorage(DEFAULT_STORAGE)
+                    .orElseThrow(() -> new UnableProcessException("No default storage defined"));
```

Comment:
@v-stepanov I will check how this method was created and used. Maybe it is some artifact from rebase and can be safely removed